src/ianitor/args_parser.py

```diff
@@ -143,6 +143,7 @@ def get_parser():
 
     parser.add_argument(
         "--port",
+        type=int, default=0,
```

Comment:
0 to 1023 are "well known ports" but there is no document that says 0 cannot be used and it has no official protocol assigned for neither TCP nor UDP. 0 is sometimes used (unofficialy) for specifying system allocated ports